torchbox/torchbox.com
# Light mode map

Follows on from https://github.com/torchbox/torchbox.com/pull/276

### Description of Changes Made

Updates to the earth version of the map for dark and light mode.

### How to Test

Test a team page on your local build, and switch between light and dark mode to see both versions of the map

### Screenshots

<details>
  <summary>Expand to see more</summary>

Note that the page will use the earth theme on the live site

![Screenshot 2024-07-05 at 12 45 11](https://github.com/torchbox/torchbox.com/assets/771869/1c9a71d9-bfad-4ac8-bf33-845a96363b62)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
